data/getTimeseries.js

```diff
@@ -0,0 +1,92 @@
+const {
+  DATA_FILE_PATH,
+  TIMESERIES_SHEET,
+  GOAL_TIMESERIES,
+  GOAL_TIMESERIES_SHEET,
+} = require("./config");
+const { excelToJson, roundNumber } = require("@sdgindex/data/parse");
+
+// Some indicators' timeseries use a different ID because they are based on
+// modeled data
+const getTimeseriesSlug = (indicator) => {
+  if (indicator.hasModeledTimeseries)
+    return indicator.modeledTimeseriesMetadata.slug;
+
+  return indicator.id;
+};
+
+const getTimeseries = ({ regions, assessments }) => {
+  // to JSON
+  const dataRows = excelToJson({
+    file: DATA_FILE_PATH,
+    sheet: TIMESERIES_SHEET,
+  });
+  const goalDataRows = excelToJson({
+    file: GOAL_TIMESERIES,
+    sheet: GOAL_TIMESERIES_SHEET,
+  });
+
+  // Identify trend + trend OECD indicators
+  const trendIndicators = assessments.filter(
+    (assessment) => assessment.type === "indicator" && assessment.hasTrend
+  );
+  const goals = assessments.filter((assessment) => assessment.type === "goal");
+
+  // Collect timeseries
+  const timeseries = [];
+
+  // Extract one timeseries for each region...
+  regions.forEach((region) => {
+    const dataForRegion = dataRows.filter((row) => row.id === region.id);
+    const goalDataForRegion = goalDataRows.filter(
+      (row) => row.id === region.id
+    );
+
+    // Sort by year, ascending
+    // dataForRegion.sort((a, b) => parseInt(a.year) - parseInt(b.year));
+
+    // add  Total
+    if (goalDataForRegion.length > 0) {
+      const TotalValues = goalDataForRegion.map((row) =>
```

Comment:
Generally, we only use uppercase for JavaScript classes and React components.

```suggestion
      const totalValues = goalDataForRegion.map((row) =>
```

Reply:
done